### DISCUSSION TITLE
Tree Questions from Basic to Advance

### DISCUSSION THREAD

**Discussion by LuvkushSharma** [community/community#86999] [Repositories]
### Select Topic Area

Show & Tell

### Body

Below is my github repository from where you can get the code.

https://github.com/LuvkushSharma/Master_Tree_Using_Java

Firstly read codes from "**_Tree_Basics_**" folder. 
Secondly read codes from "**_Tree Advanced Concepts_**" folder.

And , in the end you can read the codes from **_BST_By_LB_**.

Their will be a lot of comments in the code by which you can understand the code easily.

![220px-Tree_(computer_science) svg](https://github.com/community/community/assets/121566305/1b338526-d3f1-445c-8d24-4e26b920cbd1)

image src : Wikipedia.org

Applications of Trees:

**File Systems:**  Organizing files and directories in hierarchical structures.
**Decision-Making:**  Representing decision trees for problem-solving.
**Data Storage and Retrieval:**  Indexed databases and search trees.
**Expression Parsing:**  Syntax trees in compilers and interpreters.
**Routing Algorithms:**  Network routing decisions.
**Game AI:**  Decision-making in games like chess and Go.

Upvote this repo if you like the content 🙏

**Comment by anuragjain-git** [comment]
Nice
